python/tank/commands/unregister_folders.py

```diff
@@ -315,62 +315,12 @@ def _unregister_filesystem_location_ids(self, ids, log, prompt):
             if val != "" and not val.lower().startswith("y"):
                 log.info("Exiting! Nothing was unregistered.")
                 return []
-        
+
         log.info("Unregistering folders from Shotgun...")
         log.info("")
-        
```

Comment:
This has been moved to a class method on the PathCache, right alongside the code that creates the `Toolkit_Folders_Create` entries.